contracts/multisig/src/contract/execute.rs

```diff
@@ -188,10 +188,9 @@ pub fn authorize_callers(
 ) -> Result<Response, ContractError> {
     contracts
         .iter()
-        .map(|(contract_address, chain_name)| {
+        .try_for_each(|(contract_address, chain_name)| {
             AUTHORIZED_CALLERS.save(deps.storage, contract_address, chain_name)
-        })
-        .try_collect()?;
```

Comment:
the 2024 edition compiler is stricter with `()` and `!` types and complains about potentially unsupported type conversions. We don't need to collect the output anyway, so replacing it with `try_for_each` which matches the actual intent better